File: bots/javaClient/src/main/java/cb/bots/MyAI.java
package cb.bots;

import cb.PlayState;

public class MyAI implements BotAi {
//  todo: give your bot a super duper cool name
    private String name = "YourBotName";
    private boolean fly = true;

    public MyAI() {
    }

    @Override
    public boolean play(PlayState playState) {
//      todo: put your logic code here
        return fly;
    }

    @Override
    public String getName() {
        return name;
    }
}

File: bots/pythonClient/Bots/my_ai.py
from Bots.bot_ai import BotAI
from Bots.data import PlayState


class MyAI(BotAI):
    fly = True

    def play(self, current_game_state: PlayState):
        print(self.fly)
        # todo: put your logic code here
        return self.fly

    def get_name(self):
        return self.name

    def __init__(self):
        # todo: give your bot a super duper cool name
        self.name = "YourBotName"
